src/main/java/seedu/address/storage/JsonTagTreeStorage.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.storage;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.commons.util.FileUtil;
+import seedu.address.commons.util.JsonUtil;
+import seedu.address.model.tag.ReadOnlyTagTree;
+
+
+/**
+ * Class to access TagTree data stored as a json file on the hard disk.
```

Comment:
should this be capitalised? actually should there be standardisation in how json is referred to? some documentation refer to it as JSON and some as Json.

Reply:
i am genuinely unsure about this. Perhaps you could decide and tell us? I do believe the original `JsonAddressBook` class uses json instead of JSON.

Reply:
hmm okay, i'll keep this in mind.